### PR TITLE
Banner

### PR DESCRIPTION
**Pull request by manggalang**
within responsive

### CODE REVIEW

File: app/assets/stylesheets/pages/home/_banner.scss [hunk 1]
```diff
@@ -14,7 +14,13 @@
 		left: 50%;
 		top: 50%;
 		transform: translate(-50%, -50%);
-    width: 100%;
+		width: 100%;
+
+		@media only screen and (max-width: 611px){
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_banner.scss]: what's the point of this media query? it's so useless

File: app/assets/stylesheets/pages/home/_banner.scss [hunk 2]
```diff
@@ -27,6 +33,26 @@
 		font-size: 36px;
 		margin-top: 0;
 		margin-bottom: 25px;
+
+		@media only screen and (max-width: 1024px){
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_banner.scss]: you don't have to change the font this much, just change it when it's hit 480px width

File: app/assets/stylesheets/pages/home/_banner.scss [hunk 3]
```diff
@@ -35,6 +61,23 @@
 		margin-top: 0;
 		margin-bottom: 0;
 		line-height: 1.5;
+
+		@media only screen and (max-width: 1024px){
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_banner.scss]: try to change the font when it's hit 480px

File: app/assets/stylesheets/pages/home/_banner.scss (hunk 1 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_banner.scss]: hmm sorry

**Pull request merged**